x-pack/plugins/translations/translations/fr-FR.json

```diff
@@ -42897,4 +42897,4 @@
     "xpack.serverlessObservability.nav.projectSettings": "Paramètres de projet",
     "xpack.serverlessObservability.nav.visualizations": "Visualisations"
   }
-}
```

Comment:
not sure why these translation files changes made it here

Reply:
I am not sure why these were generated. I have reverted these three files.